planner-api/src/main/java/pl/edu/agh/kuce/planner/target/persistence/TargetRepository.java

```diff
@@ -0,0 +1,11 @@
+package pl.edu.agh.kuce.planner.target.persistence;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.Collection;
+
+public interface TargetRepository extends  JpaRepository<Target, Integer>{
+    @Query("SELECT t FROM Target t WHERE t.user.id = ?1")
```

Comment:
Same as before, I'd suggest replacing this with findByUser(User user) and omitting Query annotation.